CHANGELOG.md

```diff
@@ -125,6 +125,11 @@
 
 #### Enhancements
 
+* Adds `access_control_setter_spacing` rule to enforce no trailing whitespace between attribute names and parentheses, ensuring compatibility with Swift 6, where this spacing causes compilation errors.
+  [aryamansharda](https://github.com/aryamansharda)
+  [#5667](https://github.com/realm/SwiftLint/issues/5667)
+* Linting got around 20% faster due to the praisworthy performance
+  improvements done in the [SwiftSyntax](https://github.com/apple/swift-syntax)
```

Comment:
Please append two spaces to the end of the description. They are required to enforce a line break in the rendered Markdown.

```suggestion
* Add new `access_control_setter_spacing` rule to enforce no trailing whitespace between
  attribute names and parentheses, ensuring compatibility with Swift 6, where this spacing
  causes compilation errors.  
  [aryamansharda](https://github.com/aryamansharda)
  [#5667](https://github.com/realm/SwiftLint/issues/5667)

```